<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baseCropDialog</class>
 <widget class="QDialog" name="baseCropDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>裁剪</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>左上角</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>右下角</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="dialogBtnBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBoxX2">
     <property name="minimum">
      <number>-4000</number>
     </property>
     <property name="maximum">
      <number>4000</number>
     </property>
     <property name="value">
      <number>600</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBoxX1">
     <property name="minimum">
      <number>-4000</number>
     </property>
     <property name="maximum">
      <number>4000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinBoxY1">
     <property name="minimum">
      <number>-3000</number>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBoxY2">
     <property name="minimum">
      <number>-3000</number>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="value">
      <number>400</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>baseCropDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>baseCropDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
